### PR TITLE
More uniform docs in texture.py and _hog.py

### PR DESCRIPTION
**Pull request by j-adamczyk**
## Description

The description of `local_binary_pattern` [(docs)](https://scikit-image.org/docs/dev/api/skimage.feature.html#local-binary-pattern) `method` argument was non uniform, partially with bold and partially regular text. I've changed it to be exactly like `block_norm` argument of the `hog` function [(docs)](https://scikit-image.org/docs/dev/api/skimage.feature.html#hog). The choice of style was made based on the fact that both functions are from the same `feature` module.

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

All done, change is minor and unifies the existing docs style.

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: skimage/feature/texture.py [hunk 1]
```diff
@@ -293,18 +293,23 @@ def local_binary_pattern(image, P, R, method='default'):
     R : float
         Radius of circle (spatial resolution of the operator).
     method : {'default', 'ror', 'uniform', 'var'}
-        Method to determine the pattern.
+        Method to determine the pattern:
 
-        * 'default': original local binary pattern which is gray scale but not
+        ``default``
+           Original local binary pattern which is gray scale but not
```
Comment (mkcor) [inline skimage/feature/texture.py:299]:
```suggestion
           Original local binary pattern which is grayscale but not
```

File: skimage/feature/texture.py [hunk 2]
```diff
@@ -293,18 +293,23 @@ def local_binary_pattern(image, P, R, method='default'):
...
             rotation invariant.
-        * 'ror': extension of default implementation which is gray scale and
+        ``ror``
+           Extension of default implementation which is gray scale and
```
Comment (mkcor) [inline skimage/feature/texture.py:302]:
```suggestion
           Extension of default implementation which is grayscale and
```

File: skimage/feature/texture.py [hunk 3]
```diff
@@ -293,18 +293,23 @@ def local_binary_pattern(image, P, R, method='default'):
...
             rotation invariant.
-        * 'uniform': improved rotation invariance with uniform patterns and
-            finer quantization of the angular space which is gray scale and
+        ``uniform``
+           Improved rotation invariance with uniform patterns and finer
+            quantization of the angular space which is gray scale and
```
Comment (mkcor) [inline skimage/feature/texture.py:306]:
```suggestion
            quantization of the angular space which is grayscale and
```

File: skimage/feature/texture.py [hunk 4]
```diff
@@ -293,18 +293,23 @@ def local_binary_pattern(image, P, R, method='default'):
...
             rotation invariant.
-        * 'nri_uniform': non rotation-invariant uniform patterns variant
-            which is only gray scale invariant [2]_.
-        * 'var': rotation invariant variance measures of the contrast of local
+        ``nri_uniform``
+           Non rotation-invariant uniform patterns variant which is
+            only gray scale invariant [2]_.
```
Comment (mkcor) [inline skimage/feature/texture.py:310]:
```suggestion
            only grayscale invariant [2]_.
```

File: skimage/feature/texture.py [hunk 5]
```diff
@@ -293,18 +293,23 @@ def local_binary_pattern(image, P, R, method='default'):
...
+        ``var``
+           Rotation invariant variance measures of the contrast of local
             image texture which is rotation but not gray scale invariant.
```
Comment (mkcor) [inline skimage/feature/texture.py:313]:
```suggestion
            image texture which is only rotation invariant.
```

**Review by mkcor — APPROVED** [review]
Good catch! Thank you for this standardization effort.

**Comment by stefanv** [comment]
@mkcor Do you have a minute to review?

File: skimage/feature/texture.py [hunk 6]
```diff
@@ -293,18 +293,23 @@ def local_binary_pattern(image, P, R, method='default'):
     R : float
         Radius of circle (spatial resolution of the operator).
     method : {'default', 'ror', 'uniform', 'var'}
```
Comment (mkcor) [inline skimage/feature/texture.py:295]:
```suggestion
    method : {'default', 'ror', 'uniform', 'nri_uniform', 'var'}
```
According to what follows...

File: skimage/feature/texture.py [hunk 7]
```diff
@@ -293,18 +293,23 @@ def local_binary_pattern(image, P, R, method='default'):
...
-            finer quantization of the angular space which is gray scale and
+        ``uniform``
+           Improved rotation invariance with uniform patterns and finer
+            quantization of the angular space which is gray scale and
             rotation invariant.
-        * 'nri_uniform': non rotation-invariant uniform patterns variant
-            which is only gray scale invariant [2]_.
-        * 'var': rotation invariant variance measures of the contrast of local
+        ``nri_uniform``
+           Non rotation-invariant uniform patterns variant which is
```
Comment (mkcor) [inline skimage/feature/texture.py:309]:
```suggestion
           Non rotation-invariant uniform pattern variant which is
```

File: skimage/feature/texture.py [hunk 8]
```diff
@@ -293,18 +293,23 @@ def local_binary_pattern(image, P, R, method='default'):
...
-        * 'ror': extension of default implementation which is gray scale and
+        ``ror``
+           Extension of default implementation which is gray scale and
             rotation invariant.
-        * 'uniform': improved rotation invariance with uniform patterns and
-            finer quantization of the angular space which is gray scale and
+        ``uniform``
+           Improved rotation invariance with uniform patterns and finer
+            quantization of the angular space which is gray scale and
             rotation invariant.
```
Comment (mkcor) [inline skimage/feature/texture.py:307]:
```suggestion
            rotation-invariant.
```

**Comment by mkcor** [comment]
@stefanv sure! I guess I'll commit my own suggestions, unless @j-adamczyk shows up in the next few hours... :wink:

**Comment by mkcor** [comment]
@j-adamczyk's fork doesn't exist anymore, and this PR is not mergeable (branch has conflict), so I'll rework it in a new PR, retaining credits to the original author.

**Pull request closed without merging**